apps/ntag/arduinoNtag/NDEF/Due.h

```diff
@@ -0,0 +1,25 @@
+// redefine some stuff so code works on Due
+// http://arduino.cc/forum/index.php?&topic=153761.0
+
+#ifndef Due_h
```

Comment:
It looks like this file is effectively empty.  What's going on here?

Reply:
This file is included by arduinoNtag/NDEF/MifareClassic.h besides NdefRecord.h and I had no intention of touching that code for now. If somebody wants to use other MifareClassic they will need it. But its unlikely on yellow, so I will remove it.